### ISSUE TITLE
Internal server error only in INBOX of one account

### ISSUE THREAD

**Issue by jochenpee** [roundcube/roundcubemail#5870]
Dear developers,
this happens since a couple of months and is quite annoying to me, rendering roundcube useless for me as you can imagine.
So when I open my INBOX it says: An error occured! Server error (Internal Server Error). Other folders are OK. I upgraded to 1.2.5, error still there. 
I also tried english UI, nothing changed. I emptied my INBOX, with the first new e-mail, error reoccured.
Machine is running wheezy, dovecot 2.1.7, postfix 2.9.6, apache 2.2.22. Plugins are carddav 1.1.0, filesystem_attachments 1.0, filters 2.1.4 and jqueryui 1.10.4.
In which error log shall I look? Which logging shall I increase?
On the internet I only find old bugs which have been solved.
Please advise.

**Comment by alecpl** [comment]
Try with disabled plugins, especially those external.

**Comment by alecpl** [comment]
And look for javascript errors in browser console.

**Comment by jochenpee** [comment]
Thanks, the filters plugin was the problem. JS console says: "The character encoding of a framed document was not declared. The document may appear different if viewed without the document framing it."
Still do not know why this only happens with one user. Also I did not change or activate those plugins, didn't change the config myself. I just installed rc once and it ran nicely until this error showed up.
